pokedex/src/components/bigCard/styles.module.scss

```
@@ -0,0 +1,54 @@
+.containBigCard {
+  background-color: black;
```

Importaría los colores y usaría la variable:

```suggestion
  background-color: $black;
```

---

listo